src/NuGet.CommandLine.StagingExtensions/StageManagementResource.cs

```diff
@@ -142,6 +142,104 @@ public StageManagementResource(string stageServiceUri, HttpSource httpSource)
             return result;
         }
 
+        public async Task<StageDetailedView> Get(string id, Common.ILogger log)

```

Comment:
in general, you are right, but in this case I'm following a general pattern of creating resources in NuGet.Core, so would like to keep the code simple and matching

---
In reply to: [78220046](https://github.com/NuGet/NuGet.Staging/pull/73#discussion_r78220046) [](ancestors = 78220046)